### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Bornittah**
**Implementation**

- Add UI style

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -5,16 +5,25 @@
 
 body {
   padding: 24px;
+  background-color: #f5f6fa;
 }
 
 main {
-  margin: 12px 0;
+  /* margin: 12px 0;
+  background-color: #fafafa; */
```
Comment (Ismailco) [inline src/App.css]: - [ ] kindly consider removing all unused commented code, it makes your code messy and cluttered

File: src/App.css [hunk 2]
```diff
@@ -5,16 +5,25 @@
 
 body {
   padding: 24px;
+  background-color: #f5f6fa;
 }
 
 main {
```
Comment (Ismailco) [inline src/App.css]: - [ ] For CSS best practices kindly use class to style elements instead of naming tags

File: src/App.jsx [hunk 3]
```diff
@@ -15,8 +15,8 @@ function App() {
     <Router>
```
Comment (Ismailco) [inline src/App.jsx:15]: - [ ] [OPTIONAL] Your App is not working properly. when adding a new book the app crashes, this should be fixed in the previous steps, kindly fix this so your app is complete 

Screenshot_20220723_154923

File: src/App.jsx [hunk 4]
```diff
@@ -4,8 +4,8 @@ import {
   Routes,
   Route,
 } from 'react-router-dom';
-import Book from './components/Book';
-import Books from './components/Books';
+import BooksList from './components/BooksList';
+import BooksLibrary from './components/BooksLibrary';
 import Categories from './components/Categories';
 import NavBar from './components/shared/NavBar';
 import './App.css';
```
Comment (Ismailco) [inline src/App.jsx:11]: - [ ] [OPTIONAL] Your app is running with winnings, kindly fix them  so the app can run normally without errors or warnings, this is highly recommended 

Screenshot_20220723_154833

File: src/App.css [hunk 5]
```diff
@@ -5,16 +5,25 @@
 
 body {
```
Comment (Ismailco) [inline src/App.css]: - [ ] Your app is not styled correctly, kindly consider making the app look exactly like the original design

|Your design|Zeplin design|
|-|-|
|Screenshot_20220723_154852|bookstore (1)|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Bornittah, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Giflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Bornittah,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
appr

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**